spec/spec_helper.rb

```diff
@@ -92,6 +92,7 @@
 
     DatabaseCleaner.start
     ActionMailer::Base.deliveries.clear
+    RequestStore.clear!
```

Comment:
`RequestStore` doesn't have any ties to the setting of `config.cache_store`, so it will store entries regardless of the test environment. Since it's a simple hash, purging it before specs is the easiest / robust solution.